### DISCUSSION TITLE
Add "starred projects" to https://github.com/username/repo/projects

### DISCUSSION THREAD

**Discussion by alexesca** [community/community#3726] [Other Feature Feedback, Questions, & Ideas]
**Goal**

It would be nice to be able to "star" projects to make them stand out or find them quickly. 

**Justification**

I am starting to accumulate more open projects and it is hard to find the ones that have a biggest priority. I could sort them by activity, but still low priority projects would show up at the top. 

I am really good a keeping my projects organized and closing them on time. Love the tracking bar, and being able to star them would make it more pleasing.

Hope more people join in to get this feature through. 

I can make the first pull request if needed.

**Comment by Ara041285** [comment]
> **Goal**
> 
> It would be nice to be able to "star" projects to make them stand out or find them quickly.
> 
> **Justification**
> 
> I am starting to accumulate more open projects and it is hard to find the ones that have a biggest priority. I could sort them by activity, but still low priority projects would show up at the top.
> 
> I am really good a keeping my projects organized and closing them on time. Love the tracking bar, and being able to star them would make it more pleasing.
> 
> Hope more people join in to get this feature through.
> 
> I can make the first pull request if needed.
